### PR TITLE
Linux CI optimization

### PR DESCRIPTION
**Pull request by choulily2**
Optimazation for Linux CI

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
         id: release_please
         with:
           release-type: simple
-          pull-request-title-pattern: 'chore${scope}: 🔖 release${component} ${version}'
+          pull-request-title-pattern: "chore${scope}: 🔖 release${component} ${version}"
```
Comment (AnzhiZhang) [inline .github/workflows/release.yml]:
```suggestion
          pull-request-title-pattern: 'chore${scope}: 🔖 release${component} ${version}'
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v3
         with:
-          python-version: '3.10'
+          python-version: "3.10"
```
Comment (AnzhiZhang) [inline .github/workflows/release.yml]:
```suggestion
          python-version: '3.10'
```

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -65,7 +65,7 @@ jobs:
...
```
Comment (AnzhiZhang) [inline .github/workflows/release.yml]:
```suggestion
          python-version: '3.10'
```

**Review by AnzhiZhang — APPROVED** [review]
Are you sure this works well?

**Comment by choulily2** [comment]
oh auto format broke thing again:(

**Comment by choulily2** [comment]
Now fixed

**Pull request merged**